app/assets/javascripts/add_geo_field.js.coffee

```diff
@@ -0,0 +1,9 @@
+jQuery ->
+  $(".geo_bbox").hide() if $("#generic-file-location-form .geo_bbox ul.listing li > .form-group").hasClass("generic_file_nested_geo_bbox_id") is false
```

Comment:
You want the same function for .geo_point so that it is hidden by default as well.

`$(".geo_point").hide() if $("#generic-file-location-form .geo_point ul.listing li > .form-group").hasClass("generic_file_nested_geo_point_id") is false`